layers/aberdeenshire.map

```diff
@@ -0,0 +1,149 @@
+MAP
+  NAME "Aberdeenshire"
+  STATUS ON
+  EXTENT -180 -90 180 90
+  SHAPEPATH "../../layers"
+
+  OUTPUTFORMAT
+    NAME "geojson"
+    DRIVER "OGR/GEOJSON"
+    MIMETYPE "application/json; subtype=geojson"
+    FORMATOPTION "STORAGE=stream"
+    FORMATOPTION "FORM=SIMPLE"
+  END
+
+  LAYER
+    NAME "WSS"
+    METADATA
+      "wfs_title"         "WSS" ##REQUIRED
+      "wfs_srs"           "EPSG:3857 EPSG:27700" ## REQUIRED
+      "gml_include_items" "centralAssetId,siteCode" ## Optional (serves all attributes for layer)
+      "gml_featureid"     "fid" ## REQUIRED
+      "wfs_enable_request" "*"  # necessary
+      "wfs_getfeature_formatlist" "geojson" # so FMS.com cobrand lookup_site_code works
+    END
+    TYPE LINE
+    STATUS ON
+    CONNECTIONTYPE OGR
+    CONNECTION 'aberdeenshire/WSS.gpkg'
+    DATA 'WSS'
+    PROJECTION
+      "init=epsg:27700"
+    END
+  END #layer
+
+  LAYER
+    NAME "GULL"
+    METADATA
+      "wfs_title"         "GULL" ##REQUIRED
```

Comment:
Would be good if the wfs_title could say what the layer was, not just repeat the layer name, I guess this one is gullies, but eg BBSP? :)